LibreNMS/DB/Eloquent.php

```diff
@@ -41,7 +41,7 @@ public static function boot()
         if (! Laravel::isBooted() && is_null(self::$capsule)) {
             $install_dir = realpath(__DIR__ . '/../../');
 
-            Dotenv::create($install_dir)->load();
+            Dotenv::createMutable($install_dir)->load();
```

Comment:
@murrant Seems this was changed in newer dotenv, but didnt find any place to reach this code

Reply:
Indeed that block might be dead.  I'm not 100% sure of that.